test/measurement_calibration/generate_data.py

```diff
@@ -0,0 +1,226 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2019, 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+Generate data for measurement calibration fitter tests
+"""
+
+import os
+import json
+from test.utils import convert_ndarray_to_list_in_data
+import qiskit
+from qiskit.ignis.mitigation.measurement import (CompleteMeasFitter, TensoredMeasFitter,
+                                                 complete_meas_cal, tensored_meas_cal,
+                                                 MeasurementFilter)
+from qiskit.providers.aer.noise import NoiseModel
+from qiskit.providers.aer.noise.errors.standard_errors import pauli_error
+
+# fixed seed for simulations
+SEED = 42
+
+
+def meas_calib_circ_creation():
+    """
+    create measurement calibration circuits and a GHZ state circuit for the tests
+
+    Returns:
+        QuantumCircuit: the measurement calibrations circuits
+        list[str]: the mitigation pattern
+        QuantumCircuit: ghz circuit with 5 qubits (3 are used)
+
+    """
+    meas_calibs, state_labels = complete_meas_cal(qubit_list=[1, 2, 3], qr=5)
+
+    # Choose 3 qubits
+    qubit_1 = 1
```

Comment:
do you want to choose here the qubits from `qubit_list`?
e.g. `qubit_1 = qubit_list[0]`